Library/ViewModels/DiscoveryViewModel.swift

```diff
@@ -7,6 +7,10 @@ import ReactiveExtensions
 import Result
 
 public protocol DiscoveryViewModelInputs {
+
+  /// Call when Recommendations setting changes on Settings > Account > Privacy > Recommendations.
+  func didChangeRecommendationsSetting()
```

Comment:
The notification uses singular `recommendationSettingChanged` (for `recommendation`) yet here we're using plural (`recommendations`)...would you mind making these consistent?